extensions/python/src/main/java/com/hazelcast/jet/python/PythonServiceConfig.java

```diff
@@ -261,4 +271,32 @@ private static String requireNonBlank(@Nonnull String in, @Nonnull String name)
         }
         return in;
     }
+
+    /**
+     * Returns the channel function, see {@link #setChannelFn}.
+     *
+     * @since 5.2
+     */
+    @Nonnull
+    public BiFunctionEx<String, Integer, ? extends ManagedChannelBuilder<?>> channelFn() {
+        return channelFn;
+    }
+
+    /**
+     * Sets the channel function. The function receives a host+port tuple, and
+     * it's supposed to return a configured instance of {@link
+     * ManagedChannelBuilder}. You can use this to configure the channel, for
+     * example to configure the maximum message size etc.
+     * <p>
+     * The default value is {@code NettyChannelBuilder::forAddress}.
+     *
+     * @since 5.2
+     */
+    @Nonnull
+    public PythonServiceConfig setChannelFn(
+            @Nonnull BiFunctionEx<String, Integer, ? extends ManagedChannelBuilder<?>> channelFn
+    ) {
+        this.channelFn = requireNonNull(channelFn, "channelFn");
```

Comment:
`Preconditions.isNotNull(channelFn, "channelFn")`, you will get much cleaner message (also can be static imported btw)